contracts/build.rs

```diff
@@ -93,7 +96,7 @@ fn main() {
             .add_network(
                 "1",
                 Network {
-                    address: addr("0x3328f5f2cEcAF00a2443082B657CedEAf70bfAEf"),
+                    address: addr("0x9008D19f58AAbD9eD0D60971565AA8510560ab41"),
                     deployment_information: Some(tx(
                         "34b7f9a340e663df934fcc662b3ec5fcd7cd0c93d3c46f8ce612e94fff803909",
                     )),
```

Comment:
I think the deployment tx needs to be updated too. Same for the other address changes below.

Reply:
Oops! I forgot.